Fixing jmx references (#4019)
DataDog/documentation: content/integrations/faq/i-have-a-matching-bean-for-my-jmx-integration-but-nothing-on-collect.md

Trouble setting up your JMX integration ? Here are a few great articles to get you started :

* [Troubleshooting JMX integration][1]
* [View jmx data in jConsole and set up your jmx.yaml to collect them][2]
* [jmx.yaml error: Include Section][3]
* [Collecting Composite type JMX attributes][4]
* [How to run JMX commands in Windows?][5]

> Now, if everything is set up properly as described in the above articles, and your metric appears in [Agent log file][6] but not in the [status command][1], there is probably an issue with the `metric_type` your using.

Reviewer: ```suggestion
If everything is set up properly as described in the above articles, *and* your metric appears in the [Agent log file][6] (but *not* in the [status command][1]), then there is probably an issue with the `metric_type` that you're using.
```

## How can I solve this?

> Go to you [Agent Log file][6] and search for errors similar to these ones :

Reviewer: ```suggestion
Go to your [Agent Log file][6] and search for errors similar to the following:
```

```
2016-12-05 03:08:33,261 | WARN | JMXAttribute | Unable to get metrics from Hadoop:service=HBase,name=Master,sub=Server - tag.isActiveMaster
java.lang.NumberFormatException: For input string: "false"
 [...]
```

This means that your `Hadoop:service=HBase,name=Master,sub=Server - tag.isActiveMaster` is returning string values.

Check your `jmx.yaml` file, the following excerpt should show something similar:

{{< img src="integrations/faq/jmx_conf.png" alt="jmx_conf" responsive="true" >}}

The `java.lang.String` metric_type confirms the issue you were seeing in the logs.

> To resolve this issue, change the associated metric_type, and have your `jmx.yaml` look like this :

Reviewer: ```suggestion
To resolve this issue, change the associated metric_type, and ensure that your `jmx.yaml` file has the following configuration:
```
